### ISSUE TITLE
The weighted sum of inputs to a neuron isnt saved

### ISSUE THREAD

**Issue by willayy** [willayy/SNNPy#3]
As the title states the weighted sum of inputs to a neuron in the neural network isnt saved during the forward propogation phase. When the sum has been computed it is directly put into the sigmoid function and the output of the function is put in the neurons single value slot.

Storing the weighted sum would be more effecient in the backpropogation phase than the computationally much more expensive sigmoid inverse (what is currently used to get the weighed sum).

**Comment by willayy** [comment]
Fixed by adding neuronValue vector separately to neuronActivation vector
